features/react-native-fetch.feature

```diff
@@ -0,0 +1,200 @@
+Feature: React native source map fetch mode
+  Scenario Outline: Basic success case
+    When I start the service "react-native-<version>-bundler"
+    And I wait for 2 seconds
```

Comment:
I assume you probably checked, but is there some output or state you could wait for, rather than an arbitrary timeout?

Reply:
Possibly, I meant to ask Steve — waiting for the bundle URL didn't work so I assumed not